Windows CI tests to disable.

I tried to disable the test on windows that was causing problems in #1963 , but it looks like I missed two that were failing slightly less frequently:

```
=========================== short test summary info ===========================
FAILED test/test_cli.py::TestCLI::test_update_flags - aiohttp.client_exceptio...
FAILED test/test_cvedb.py::TestCVEDB::test_refresh_nvd_json - rich.errors.Liv...
======= 2 failed, 19 passed, 1 skipped, 30 warnings in 90.32s (0:01:30) =======
```

As before: these tests are running fine on linux and the code is the same on windows. I'm not sure why they're failing only on windows. 

It *looks* like network failure connecting to NVD or possible hitting a rate limit, although I wouldn't expect the latter to be an issue when we're using an NVD_API_KEY on the main repo.  It may only happen when we have multiple things merged at once so jobs are running back to back, but I'd expect to sometimes see failures on linux if that was the problem so I just don't know.

But NVD clearly doesn't want us to hit it too often so it's probably more polite to do these tests only once anyhow.

So basically, in the interest of having more stable CI I'd like to disable these two tests on windows only and we'll see if that solves the problem for the moment.

--- a/test/test_cli.py
+++ b/test/test_cli.py
@@ -154,6 +154,9 @@ class TestCLI(TempDirTest):
         assert e.value.args[0] == ERROR_CODES[SystemExit]
 
     @pytest.mark.skipif(LONG_TESTS() != 1, reason="Update flag tests are long tests")
+    @pytest.mark.skipif(
+        sys.platform == "win32", reason="Causing failures in CI on windows only"
+    )
     def test_update_flags(self):
         assert (
             main(["cve-bin-tool", "-x", "-u", "never", "-n", "json", self.tempdir]) != 0

--- a/test/test_cvedb.py
+++ b/test/test_cvedb.py
@@ -3,6 +3,7 @@
 
 import datetime
 import shutil
+import sys
 import tempfile
 from test.utils import LONG_TESTS
 
@@ -27,6 +28,9 @@ class TestCVEDB:
     @pytest.mark.asyncio
     @pytest.mark.skipif(
         LONG_TESTS() != 1, reason="Skipping NVD calls due to rate limits"
+    )
+    @pytest.mark.skipif(
+        sys.platform == "win32", reason="Causing failures in CI on windows only"
     )
     async def test_refresh_nvd_json(self):
         await self.cvedb.refresh()

